beacon-chain/sync/metrics.go

```diff
@@ -86,7 +86,7 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "block_arrival_latency_milliseconds",
 			Help:    "Captures blocks propagation time. Blocks arrival in milliseconds distribution",
-			Buckets: []float64{250, 500, 1000, 1500, 2000, 4000, 8000, 16000},
+			Buckets: []float64{100, 500, 1000, 4000, 8000, 12000},
```

Comment:
I think you need even more granularity around 500 to 1000ms. Here's a current screenshot from mainnet metrics 

![Screenshot from 2022-09-12 09-17-20](https://user-images.githubusercontent.com/7246818/189677759-ac51924e-9fa6-4cc7-965c-416658f8971f.png)

Reply:
Thanks for the chart! I added 250 and 750 back

Reply:
OK great. Are you sure you want to remove 1500 and 2000? That seems like common latency buckets

Reply:
No, you are right. I was worried that too many buckets might be bad, but then I realized there's not much downside to it. Thanks for the suggestions!